test/integration/apiserver/apiserver_test.go

```diff
@@ -99,7 +99,7 @@ func newRS(namespace string) *v1beta1.ReplicaSet {
 	}
 }
 
-var cascDel string = `
+var cascDel = `
```

Comment:
I tried this way, but I found that golang doesn't support it.
Short variable declarations may appear only inside functions.
Reference:
https://golang.org/ref/spec#Short_variable_declarations
Thanks